lib/actions-oclif/os/versions.ts

```diff
@@ -54,13 +59,41 @@ export default class OsVersionsCmd extends Command {
 	public async run() {
 		const { args: params } = this.parse<FlagsDef, ArgsDef>(OsVersionsCmd);
 
-		const {
-			versions: vs,
-			recommended,
-		} = await getBalenaSdk().models.os.getSupportedVersions(params.type);
+		let versions = await this.getOsVersionsForDeviceType(params.type);
+		if (versions.length === 0) {
```

Comment:
If a device type "alias" such as `'nuc'` is used, `getOsVersionsForDeviceType` returns an empty list. If this is the case, fetch the aliases, resolve the actual device type name (e.g. `'intel-nuc'`), and call `getOsVersionsForDeviceType` again for the actual device type name. Querying and computing the list of aliases is relatively slow/expensive, which is why I thought it should only be done if the list of versions is empty. A note advising against the use of aliases was added to the help text.